test/schedule_manager_test.exs

```diff
@@ -28,72 +29,67 @@ defmodule Verk.ScheduleManagerTest do
   test "handle_info :fetch_retryable without jobs to retry", %{ script: script } do
     state = %State{ redis: :redis }
     now = :now
-    expect(Timex.Time, :now, [:seconds], now)
     expect(Redix, :command, [:redis, ["EVALSHA", script, 1, "retry", now]], {:ok, nil})
 
     assert handle_info(:fetch_retryable, state) == { :noreply, state }
     assert_receive :fetch_retryable
 
-    assert validate [Timex.Time, Redix]
+    assert validate [ Redix]
   end
 
-  test "handle_info :fetch_retryable with jobs to retry", %{ script: script } do
+  test "handle_info :fetch_retryable with jobs to retry", %{ script: script  } do
     state = %State{ redis: :redis }
-    now = :now
+    now = Time.now
+    expect(Time, :now, [], now)
     encoded_job = "encoded_job"
-    expect(Timex.Time, :now, [:seconds], now)
-    expect(Redix, :command, [:redis, ["EVALSHA", script, 1, "retry", now]], {:ok, encoded_job})
+    expect(Redix, :command, [:redis, ["EVALSHA", script, 1, "retry", now], state], {:ok, encoded_job})
```

Comment:
Yeah, the move back to the original code for that expectation didn't work:

```
  test "handle_info :fetch_retryable with jobs to retry", %{ script: script  } do
    state = %State{ redis: :redis }
    now = Time.now
    expect(Time, :now, [], now)
    encoded_job = "encoded_job"
    expect(Redix, :command, [:redis, ["EVALSHA", script, 1, "retry", now]], {:ok, encoded_job})

    assert handle_info(:fetch_retryable, state) == { :noreply, state }
    assert_receive :fetch_retryable, 5

    assert validate [Time, Redix]
  end
```

yields: 

```
  1) test handle_info :fetch_retryable with jobs to retry (Verk.ScheduleManagerTest)
     test/schedule_manager_test.exs:40
     ** (FunctionClauseError) no function clause matching in :gen_server.call/2
     stacktrace:
       (stdlib) gen_server.erl:204: :gen_server.call/2
       src/meck_proc.erl:108: :meck_proc.set_expect/2
       src/meck.erl:252: :meck.expect/4
       test/schedule_manager_test.exs:45: (test)
```

Any ideas what might be going on here?